recipes/acado/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+import os
+import glob
+
+from conans import ConanFile, CMake, tools
+
+
+class AcadoConan(ConanFile):
+    name = "acado"
+    description = "ACADO Toolkit is a software environment and algorithm collection for automatic control and dynamic optimization."
+    license = "LGPL-3.0"
+    topics = ("conan", "acado", "control", "optimization", "mpc")
+    homepage = "https://github.com/acado/acado"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "cmake/qpoases.cmake"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "examples": [True, False],
+        "testing": [True, False],
+        "developer": [True, False],
+        "internal": [True, False],
+        "codegen_only": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "examples": False,
+        "testing": False,
+        "developer": False,
+        "internal": False,
+        "codegen_only": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob("acado-*/")[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+
+        self._cmake.definitions["ACADO_BUILD_SHARED"] = self.options.shared
+        self._cmake.definitions["ACADO_BUILD_STATIC"] = not self.options.shared
+
+        self._cmake.definitions["ACADO_WITH_EXAMPLES"] = self.options.examples
+        self._cmake.definitions["ACADO_WITH_TESTING"] = self.options.testing
+        self._cmake.definitions["ACADO_DEVELOPER"] = self.options.developer
+        self._cmake.definitions["ACADO_INTERNAL"] = self.options.internal
```

Comment:
See previous comments

```suggestion
        self._cmake.definitions["ACADO_WITH_EXAMPLES"] = False
        self._cmake.definitions["ACADO_WITH_TESTING"] = False
        self._cmake.definitions["ACADO_DEVELOPER"] = False
        self._cmake.definitions["ACADO_INTERNAL"] = False
```

Reply:
done